src/site/content/en/blog/fetch-metadata/index.md

```diff
@@ -50,7 +50,7 @@ Fetch Metadata request headers are a new web platform security feature designed
 Requests originating from sites served by your own server (same-origin) will continue to work.
 {% endCompareCaption %}
 
-![A fetch request from https://site.example for the resource https://site.example/foo.json in JavaScript causes the browser to send the HTTP request header "Sec Fetch-Site: same-origin".](same-origin-request.png)
+{% Img src="image/admin/aRsy2xULTR4TM2sMMsbQ.png", alt="A fetch request from site.example for the resource site.example/foo.json in JavaScript causes the browser to send the HTTP request header 'Sec Fetch-Site: same-origin'.", width="800", height="176" %}
```

Comment:
just curious if you want to leave the `https://` protocol in here. I don't think it materially changes the article but wanted to note that it is a slight tweak.

Reply:
The markdown linter doesnt like urls that arent in blocks or tags, so something in a shortcode fails the linting test. Removing the protocol lets it pass.

Reply:
oh you can use a comment to disable that on a per file basis. I'm not sure what the rule is called but here's an example using the duplicate heading rule

```md
<!--lint disable no-duplicate-headings-->
{% Img src="image/admin/aRsy2xULTR4TM2sMMsbQ.png", alt="A fetch request from site.example for the resource site.example/foo.json in JavaScript causes the browser to send the HTTP request header 'Sec Fetch-Site: same-origin'.", width="800", height="176" %}
<!--lint enable no-duplicate-headings-->
```